hack/kind/new-cluster.sh

```diff
@@ -30,14 +30,16 @@ EOF
 
 # Make sure the local Docker image registry is connected to the same network
 # as the kind cluster
-docker network connect "kind" "${reg_name}" || true
+if ! docker network inspect kind >/dev/null 2>&1; then
```

Comment:
I think this may be the added piece that may be tripping me up... for instance if the `kind` docker network persists but I have created a new kind cluster and registry... with this logic, the necessary `docker network connect` command would be skipped.

Reply:
Ya... if it's a new registry and existing network, this leads to a problem.

Reply:
I fixed this, signed the commit, and force pushed.